apps/test/integration/levelSolutions/applab2/ec_setprop.js

```diff
@@ -153,18 +153,33 @@ module.exports = {
       runBeforeClick: function(assert) {
```

Comment:
As written, this test assumes an additional escape when retrieving the escaped image URL from a CSS property vs when you retrieve it from an image src attribute.  While this seems to be true in PhantomJS, it's not happening in Chrome or Firefox.

Since the spirit of the test seems to be checking that appropriate escapes are applied at all, I'm loosing the assertions to accept either escaped version, so that the test will pass on both Chrome and Phantom.